native/cocos/renderer/gfx-validator/DeviceValidator.h

```diff
@@ -106,6 +107,7 @@ class CC_DLL DeviceValidator final : public Agent<Device> {
 
     bool _recording{false};
     uint64_t _currentFrame{1U};
+    IXRInterface *_xr{nullptr};
```

Comment:
Is IXRInterface used in `DeviceValidator`? It seems not used.

Reply:
doesn't used, can be removed